docs/pages/machine-id/guides/hostcertificate.mdx

```diff
@@ -0,0 +1,364 @@
+---
+title: Host Certificates with Machine ID
+description: Issuing Host Certificates with Machine ID for OpenSSH servers.
+---
+
+In this guide, you will learn how to create host certificates through Machine ID to allow Teleport users (manually or automated through Machine ID) to access OpenSSH servers. This allows for the benefits of
```

Comment:
I wonder if it's overcomplicating things to mention "manually or automated through Machine ID". Perhaps it makes more sense that this guide improves on https://goteleport.com/docs/server-access/guides/openssh/ by ensuring that certificates are rotated regularly without manual intervention.

Reply:
I agree with this pending any differing opinions, intro is being reworked to better reflect this